### DISCUSSION TITLE
垂类模型下载 无法加群，请问有群号吗

### DISCUSSION THREAD

**Discussion by BTBU2018** [PaddlePaddle/PaddleOCR#12429] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
![50836be8-54eb-4047-acf8-ed520e12d10e](https://github.com/PaddlePaddle/PaddleOCR/assets/30739893/3939da1e-9f14-4dc5-a68d-415671a8e02c)

**Comment by zrp2023** [comment]
same problem

**Comment by minushuang** [comment]
通问，模型哪里可以下载
